Makefile

```diff
@@ -125,16 +188,23 @@ safe:
 demo-sidecar:
 	./hack/install-workload-using-sidecar.sh
 
+demo-sidecar-local:
+	./hack/install-workload-using-sidecar-local.sh
+
 # Installs the demo app to play with.
 demo-sdk:
 	./hack/install-workload-using-sdk.sh
 
-demo-sidecar-local:
-	./hack/install-workload-using-sidecar-local.sh
-
 demo-sdk-local:
 	./hack/install-workload-using-sdk-local.sh
 
+# Install the demo app that uses init container.
+demo-init-container:
+	./hack/install-workload-using-init-container.sh
```

Comment:
A script better belongs to a script file.

Using Makefile tasks as mini script files was turning out to be cumbersome.

Passing arguments to scripts from the Makefile works better.